platform/android/doc_classes/EditorExportPlatformAndroid.xml

```diff
@@ -380,6 +380,9 @@
 		<member name="permissions/persistent_activity" type="bool" setter="" getter="" deprecated="Deprecated in API level 15.">
 			Allow an application to make its activities persistent.
 		</member>
+		<member name="permissions/post_notifications" type="bool" setter="" getter="" deprecated="">
```

Comment:
I just noticed the `deprecated=""` which was likely copied from one of the surrounding errors. This will mark this entry as deprecated in the docs, when that's not intended.

Reply:
Bah, my bad, thanks for noticing. I'll remove it in another PR - I believe a few other permission properties could use an update to their description...

Reply:
Fixed in https://github.com/godotengine/godot/pull/90546